src/components/Favorites.js

```diff
@@ -1,13 +1,33 @@
 import React from 'react';
-// import '../styles/Main.css';
+import '../styles/Favorites.css';
+
+const myFavorites = [];
+export const addFavorite = (nameP, challenge_rating, indexP) => {
+  if (myFavorites.length === 0) {
+    myFavorites.push({ name: nameP, index: indexP, CR: challenge_rating, key: indexP })
+  }
+  if (!myFavorites.find(creature => creature.name.includes(nameP))) {
+    myFavorites.push({ name: nameP, index: indexP, CR: challenge_rating, key: indexP })
+  }
```

Comment:
This favorites component is really just a stretch goal that I wanted to add.  The above code (lines 4-10) are working to eliminate the problem of a user submitting the same creature multiple times to the fav list, but I'm not sure it's the most elegant way.

Reply:
Good question! I wonder if you could have state in App that holds the favorites and the addFavorite method would live inside App as well... that way you could pass the method down to `CreatureDetails` and the favorites array to `Favorites` as props.